Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorGenerateSnapshot.cpp

```diff
@@ -19,7 +19,7 @@ using namespace improbable;
 namespace
 {
 
-using NameToEntityIdMap = worker::Map<std::string, worker::EntityId>;
+using PathNameToEntityIdMap = worker::Map<std::string, worker::EntityId>;
```

Comment:
Sorry, do you think this could be named to `PathToEntityIdMap`? I don't know the difference between a Path and a Path Name.

Reply:
I don't think there is,  the reason why I suggested PathName was because unreals api for getting the path is called `GetPathName`.